internal/common/conversion/type_conversion.go

```diff
@@ -56,3 +56,17 @@ func IsStringPresent(strPtr *string) bool {
 func MongoDBRegionToAWSRegion(region string) string {
 	return strings.ReplaceAll(strings.ToLower(region), "_", "-")
 }
+
+func SlicePtrToSlice[T any](v *[]T) []T {
```

Comment:
can we please unit test these two methods?

Reply:
added